tikv-client/src/main/java/com/pingcap/tikv/codec/MyDecimal.java

```diff
@@ -16,12 +16,13 @@
 package com.pingcap.tikv.codec;
 
 import com.google.common.annotations.VisibleForTesting;
+import java.io.Serializable;
 import java.math.BigDecimal;
 import java.util.Arrays;
 
 // TODO: We shouldn't allow empty MyDecimal
 // TODO: It seems MyDecimal to BigDecimal is very slow
-public class MyDecimal {
+public class MyDecimal implements Serializable {
```

Comment:
Any reason to implements `Serializable`?

Reply:
`MyDecimal` Type data is used in `RDD` and is serialized by spark into `Task`.

Reply:
So other data types also require implementing `Serializable`?

Reply:
which class do you mean?

Reply:
I misunderstood the "data type"